charts/fission-all/templates/deployment.yaml

```diff
@@ -128,7 +128,7 @@ spec:
     spec:
       containers:
       - name: controller
-        image: "{{ .Values.repository }}/{{ .Values.image }}:{{ .Values.imageTag }}"
+        image: {{ template "fission-all-bundleImage" . }}
```

Comment:
From Helm documentation

> It is considered preferable to use `include` over `template` in Helm templates simply so that the output formatting can be handled better for YAML documents.

https://helm.sh/docs/chart_template_guide/#the-include-function

Reply:
Replaced template with include.